src/redirect.js

```diff
@@ -51,43 +52,43 @@ function setPassportTableData(tokenData) {
         },
         {
             tableRowId: 'tableUserRegistered',
-            tableRowData: Date(tokenData.bindid_network_info?.user_registration_time)
+            tableRowData: tokenData.bindid_network_info?.user_registration_time
```

Comment:
@shachartransmit is this not using `unixToDate`. on purpose?

Reply:
@darkreiser Yes, this is actually a string, not a date. (e.g `Over 28 days ago`)